docs/geedocs/answer/3230746.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
 <meta charset="UTF-8"/>
-<title>Portable User Guide for Mac OS, Windows, and Linux</title>
+<title>Portable User Guide for Windows, and Linux</title>
```

Comment:
:smiley: I don't think we have a comma style for the documentation. I'd rather leave the documents as they are, and not go through looking for commas.